packages/core/js-sdk/src/admin/refund-reasons.ts

```diff
@@ -71,4 +74,154 @@ export class RefundReason {
       }
     )
   }
+
+  /**
+   * This method retrieves a refund reason by ID. It sends a request to the
+   * [Get Refund Reason](https://docs.medusajs.com/api/admin#refund-reasons_getrefundreasonsid)
+   * API route.
+   *
+   * @param id - The refund reason's ID.
+   * @param query - Configure the fields and relations to retrieve in the refund reason.
+   * @param headers - Headers to pass in the request.
+   * @refunds The refund reason's details.
```

Comment:
```suggestion
   * @returns The refund reason's details.
```
other methods in this file have the same typo